pkg/alertmanager/multitenant.go

```diff
@@ -188,8 +189,7 @@ func NewMultitenantAlertmanager(cfg *MultitenantAlertmanagerConfig, cfgCfg confi
 			cluster.DefaultProbeInterval,
 		)
 		if err != nil {
-			level.Error(util.Logger).Log("msg", "unable to initialize gossip mesh", "err", err)
-			os.Exit(1)
+			return nil, errors.Wrap(err, "unable to initialize gossip mesh")
```

Comment:
This has removed the log, which is OK as far as it's logged somewhere in the parent. However, looking at `initAlertmanager()` (`modules.go`), in case `NewMultitenantAlertmanager()` returns error we do return "success" instead of the error. I think it's a bug. May you fix it, please?

Reply:
Wow, quite the bug haha. I’ll fix it